lib/views/admin_request/hidden_user_explanation/_personal_information.text.erb

```diff
@@ -1,3 +1,20 @@
 <%= _('We consider it is not a valid FOI request as it was correspondence ' \
       'about your personal circumstances, and we have therefore hidden it ' \
       'from other users.') %>
+
+<%= _('* If you would like to know what information a public authority holds ' \
+      'about you, you should make a “Subject Access Request” in private ' \
+      'Data Protection law. The Information Commissioner’s website provides '\
+      'advice on accessing your own personal information: ' \
+      'https://ico.org.uk/your-data-matters/your-right-of-access') %>
+
+<%= _('* It includes extraneous material such as allegations, personal ' \
```

Comment:
I don't think these should be "bullet points" given they seem like an either/or option?